java-utils/java-functions

```diff
@@ -106,12 +106,21 @@ _load_java_conf()
 	JAVACMD_OPTS="${java_opts_save}"
     fi
 
-    if [ "_${JAVA_ABRT}" != "_off" -a -f "@{abrtlibdir}/libabrt-java-connector.so" -a -f "@{rundir}/abrt/abrtd.pid" ]; then
-	_log "ABRT Java connector was enabled"
-	_log "Using ABRT Java agent: @{abrtlibdir}/libabrt-java-connector.so"
-	JAVACMD_OPTS="${JAVACMD_OPTS} -agentpath:@{abrtlibdir}/libabrt-java-connector.so=abrt=on"
+    if [ "_${JAVA_ABRT}" != "_off" ]; then
+        _log "ABRT Java connector was enabled"
```

Comment:
It logs the connector is enabled even when it's not installed. The logging should be moved around a bit, so that it says "enabled" when it's actually enabled and "disabled" otherwise.